importer/load.py

```diff
@@ -5,41 +5,23 @@
 from datetime import date, datetime
 from neo4j.v1 import GraphDatabase, basic_auth
 
-# Entities defined:
-# - Article
-# - Author
-# - Country
-# - Month
-# - Publication
-# - RetractionReason
-# - Topic
-# - Year
-
-# Relationships defined:
-# - (Article) -[PUBLISHED_IN]-> (Publication)
-# - (Article) -[AUTHORED_BY]-> (Author)
-# - (Article) -[ORIGINATED_IN]-> (Country)
-# - (Article) -[DISCUSSES]-> (Topic)
-# - (Article) -[RETRACTED_FOR]-> (RetractionReason)
-# - (Article) -[PUBLISHED_IN]-> (Year)
-# - (Article) -[PUBLISHED_IN]-> (Month)
-
-# NOTE: How do we handle Python dependencies currently?
 # NOTE: Doesn't work with outbound Neo4j, just Docker
 
 
 def main(argv):
 
-    URL = 'localhost'
-    USER = 'neo4j'
-    PASS = 'neo4j'
+    # url = 'localhost'
+    # user = 'neo4j'
+    # password = 'neo4j'
 
-    if 'DATABASE_URL' in os.environ:
-        URL = os.environ['DATABASE_URL']
-    if 'DATABASE_PASS' in os.environ:
-        PASS = os.environ['DATABASE_PASS']
-    if 'DATABASE_USER' in os.environ:
-        USER = os.environ['DATABASE_USER']
+    if all(var in os.environ for
+           var in ['DATABASE_URL', 'DATABASE_USER', 'DATABASE_PASS']):
+                url = os.environ['DATABASE_URL']
+                pwd = os.environ['DATABASE_PASS']
+                usr = os.environ['DATABASE_USER']
+    else:
+        raise ValueError("Ensure environment variables DATABASE_URL, "
+                         "DATABASE_PASS and DATABASE_USER set.")
```

Comment:
Hmm, I'm not sure if `ValueError` is right here: https://docs.python.org/2/library/exceptions.html . I generally use `RuntimeError` if its a user error.